doc/braceless.txt

```diff
@@ -225,11 +225,8 @@ Default:
 *g:braceless_jump_prev_key*
 *g:braceless_jump_next_key*
 
-By default, these let you move around using the |[[| and |]]| keys.  The following
-sequences are also mapped with these keys:
-
-  *g]*    Move forward to an increased indent level
-  *g[*    Move backward to a decreased indent level
+By default you can move between block heads using the |[[| and |]]| motions.
+This fits better with VIM's traditional notion of a |section|.
```

Comment:
This could read:

```
By default you can move between block heads using the |[[| and |]]| motions.
This fits better with VIM's traditional notion of a |section|.  If you want
motions for moving between block indent levels ... etc
```